lib/shopify_api/resources/location.rb

```diff
@@ -1,4 +1,8 @@
 module ShopifyAPI
   class Location < Base
+
+    def inventory_levels
+      ShopifyAPI::InventoryLevel.find :all, from: "/admin/locations/#{id}/inventory_levels.json"
```

Comment:
Nitpick, but I prefer to consistently use parentheses on method calls.

Exceptions to that rule include declarative style method calls in Rails like "has_many :things" or `assert...` calls. Seems like we do this with the `fake` calls in this gem as well. But maybe everything else should stay consistently called with parentheses.